webrtc.html

```diff
@@ -11470,18 +11482,57 @@ <h2>
                     <p>
                       If either (but not both) of <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}}
                       and <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}} are [= map/exist | missing =],
-                      or if they both [=map/exist=] and <var>first</var>.{{RTCRtpCodec/sdpFmtpLine}}
-                      is different from <var>second</var>.{{RTCRtpCodec/sdpFmtpLine}}, return
-                      <code>false</code>.
+                      return <code>false</code>.
```

Comment:
RTP payload specs can specify default values of fmtp parameters. So it is possible for codecs to match even if a peer's sdpFmtpLine is missing. Example: In H.265, default level-id is 3.1, default profile is Main.  Are default values included in the algorthym?

Reply:
FWIW: some implementations backfill these values and simplify the life of the JS layer that way

Reply:
I have a note that says "In some cases a parameter can be omitted and still be inferred", that should cover default values